packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -438,7 +463,12 @@ export async function runCompassOnce(args: string[], timeout = 30_000) {
     args,
     timeout,
   });
-  const { stdout, stderr } = await promisify(execFile)(
+  // For whatever reason, running the compass CLI frequently completes what it
+  // was set to do and then exits the process with error code 1 and no other
+  // output. So while figuring out what's going on, don't ever throw but do log
+  // the error. Assertions that follow runCompassOnce() can then check if it did
+  // what it was supposed to do and fail if it didn't.
+  const { error, stdout, stderr } = await execFileIgnoreError(
```

Comment:
Here's an example of it happening:

https://spruce.mongodb.com/task/10gen_compass_main_ubuntu_test_packaged_app_5x_enterprise_patch_f37559b2b2a7e11b303d071e17e050865473a770_65a13f5c32f4174972d8a3cd_24_01_12_13_32_13/logs?execution=0&sortBy=STATUS&sortDir=ASC

or the logs:

search for `Ran compass with args`

https://parsley.mongodb.com/evergreen/10gen_compass_main_ubuntu_test_packaged_app_5x_enterprise_patch_f37559b2b2a7e11b303d071e17e050865473a770_65a13f5c32f4174972d8a3cd_24_01_12_13_32_13/0/task?bookmarks=0,52456&shareLine=10408


This started happening recently with the telemetry updates. My hunch is there's some timing thing there, but there's no error so it seems like there is no way to tell.